apps/web/src/styles/globals.css

```
@@ -1,11 +1,14 @@
 @import "tailwindcss";
 @import "@heroui/styles";
+@import "@heroui-pro/react/css";
+@import "@heroui-pro/react/themes/glass";
 @import "./heroui-theme.css";
 
 /* Scan route and component files for Tailwind utility classes */
 @source "../routes";
 @source "../components";
 @source "../../node_modules/@heroui/react/dist";
+@source "../../node_modules/@heroui-pro/react/dist";
```

Stylelint (17.12.0)

[error] Unexpected unknown at-rule "`@source`" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)